subsys/mgmt/settings/Kconfig

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Settings Management"
+
+config MGMT_SETTINGS
+	bool "Settings Management Module"
+	depends on MCUMGR
```

Comment:
Should it depend on `SETTINGS` and some flash subsystem as well?

Reply:
If generalized to just work with any storage then no.